src/script/entity/message/MemberMessage.ts

```diff
@@ -82,17 +102,15 @@ export class MemberMessage extends SystemMessage {
 
     // Users joined the conversation without self
     this.remoteUserEntities = ko.pureComputed(() => {
-      return this.userEntities()
-        .filter(userEntity => !userEntity.is_me)
-        .map(userEntity => userEntity);
+      return this.userEntities().filter(userEntity => !userEntity.is_me);
```

Comment:
Mapping `userEntity` to `userEntity` didn't really make sense (left over from CoffeeScript migration).